src/patients/edit/EditPatient.tsx

```diff
@@ -69,24 +70,12 @@ const EditPatient = () => {
   }
 
   const onSave = async () => {
-    await dispatch(
-      updatePatient(
-        {
-          ...patient,
-          fullName: getPatientName(patient.givenName, patient.familyName, patient.suffix),
-          index:
-            getPatientName(patient.givenName, patient.familyName, patient.suffix) + patient.code,
-        },
-        onSuccessfulSave,
-      ),
-    )
+    const newPatient = cleanupPatient(patient)
```

Comment:
let's have the redux `updatePatient` and `createPatient` do this

Reply:
✅